examples/lambda.R

```diff
@@ -0,0 +1,61 @@
+# Lambda examples
+
+#-------------------------------------------------------------------------------
+
+# Create a Lambda function package to upload.
+
+code <- 'exports.handler = async (event, context) => { return "Hello!"; };'
+path <- tempdir()
+js_file <- file.path(path, "lambda.js")
+writeLines(code, js_file)
+
+zip_file <- file.path(path, "lambda.zip")
+utils::zip(zip_file, js_file, flags = "-j")
+zip_contents <- readBin(zip_file, "raw", n = 1e5)
+
+#-------------------------------------------------------------------------------
+
+# Set up an IAM role for the Lambda function.
+
+role_policy <- list(
+  Version = "2012-10-17",
+  Statement = list(
+    list(
+      Effect = "Allow",
+      Principal = list(
+        Service = "lambda.amazonaws.com"
+      ),
+      Action = "sts:AssumeRole"
+    )
+  )
+)
+
+iam <- paws.iam::create_role(
+  RoleName = "MyRole",
+  AssumeRolePolicyDocument = jsonlite::toJSON(role_policy, auto_unbox = TRUE),
+  PermissionsBoundary = "arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole"
+)
+
+#-------------------------------------------------------------------------------
+
+# Create the Lambda function.
+paws.lambda::create_function(
```

Comment:
I get `Error: SerializationException` when running the lambda example.

Reply:
What do you have for `zip_contents`.

Reply:
This is my `zip_contents`:
```
  [1] 50 4b 03 04 0a 00 00 00 00 00 d0 ae 83 4d cd 76 69 7f 43 00 00 00 43 00 00 00
 [27] 09 00 1c 00 6c 61 6d 62 64 61 2e 6a 73 55 54 09 00 03 68 ec 05 5c 68 ec 05 5c
 [53] 75 78 0b 00 01 04 56 05 10 00 04 01 02 10 00 65 78 70 6f 72 74 73 2e 68 61 6e
 [79] 64 6c 65 72 20 3d 20 61 73 79 6e 63 20 28 65 76 65 6e 74 2c 20 63 6f 6e 74 65
[105] 78 74 29 20 3d 3e 20 7b 20 72 65 74 75 72 6e 20 22 48 65 6c 6c 6f 21 22 3b 20
[131] 7d 3b 0d 0a 50 4b 01 02 1e 03 0a 00 00 00 00 00 d0 ae 83 4d cd 76 69 7f 43 00
[157] 00 00 43 00 00 00 09 00 18 00 00 00 00 00 01 00 00 00 a4 81 00 00 00 00 6c 61
[183] 6d 62 64 61 2e 6a 73 55 54 05 00 03 68 ec 05 5c 75 78 0b 00 01 04 56 05 10 00
[209] 04 01 02 10 00 50 4b 05 06 00 00 00 00 01 00 01 00 4f 00 00 00 86 00 00 00 00
[235] 00
```

Reply:
Mine is slightly different

Reply:
if you `writeBin` it to disk, is it a valid zip file?

Reply:
I get the same error for the paws.comprehend example

Reply:
Maybe this needs stuff from `feature/rest_xml`.

Reply:
I get this error now
```
Error: InvalidParameterValueException: The role defined for the function cannot be assumed by Lambda.
```

Reply:
Can you check whether you have the role in your account?

Reply:
i checked and it was there

Reply:
Can you manually create a Lambda function with that role?

Reply:
Yes

Reply:
Do you have a value for `iam$Role$Arn`.

Reply:
yes i do

Reply:
wait it worked this time. I just undid everything, and retried it. Sorry

Reply:
Cool 👍